src/printer.js

```diff
@@ -1541,10 +1548,10 @@ function genericPrintNoParens(path, options, print) {
       return printFlowDeclaration(path, [
         "module.exports",
         path.call(print, "typeAnnotation"),
-        ";"
+        semi
       ]);
     case "DeclareVariable":
-      return printFlowDeclaration(path, ["var ", path.call(print, "id"), ";"]);
+      return printFlowDeclaration(path, ["var ", path.call(print, "id"), semi]);
```

Comment:
does anyone know if Flow *requires* semis inside declarations and other such places?